package/suse-migration-services-spec-template

```diff
@@ -61,19 +60,20 @@ BuildArch:        noarch
 %description
 Systemd services to prepare and run a distribution migration process.
 
-%package          -n %{pythons}-migration
+%package          -n python3-suse-migration-services
```

Comment:
This change looks so easy right ... and opens a can of worms and I can already smell the nightmare it will produce the next time I try to submit the DMS to SLE15. I see the obsoletes was set but no provides as documented here:

* https://en.opensuse.org/openSUSE:Upgrade_dependencies_explanation#Renaming_a_package

And again the question; What issue is this rename solving ? Yes the package is named python3-migration and oh well yes the python namespace is `suse_migration_services` and not just `migration` and by the way the package uses dashes and not underscores 

Really we are creating problems and trying to fix something that really nobody will care for. Whom does this help ?

Reply:
I really object here. The package names as they exist in SLE15 doesn't have to be changed for SLE16 unless this creates any sort of real conflict. We have to maintain this not only for SLFO and a package rename is an expensive change. I'm not even sure if we need an ECO for SLE15 in order to apply this change.

Reply:
done